src/Type/DataSchema.ts

```diff
@@ -0,0 +1,53 @@
+import SchemaProperty from './SchemaProperty';
+
+/**
+ * Class represents the schema of imported geo-data, to have information about available
+ * properties and data ranges.
+ * Comparable to a DescribeFeatureType response for an OGC WFS.
+ * This is modelled as JSON schema:
+ *
+ *  {
+ *    "title": "Person",
+ *    "type": "object",
+ *    "properties": {
+ *      "firstName": {
+ *        "type": "string"
+ *      },
+ *      "lastName": {
+ *        "type": "string"
+ *      },
+ *      "age": {
+ *        "description": "Age in years",
+ *        "type": "integer",
+ *        "minimum": 0
+ *      }
+ *   }
+ * }
+ *
+ * @type DataSchema
+ */
+type DataSchema = {
+
+  /**
+   * Optional title for the described entity
+   *
+   * @type {string}
+   */
+  title: string | '';
```

Comment:
I think it should be `title?: string;` if its optional

Reply:
By this notation I wanted to ensure that ``title`` is always a String (= default value) and not ``null`` or ``undefined``.  Does this make sense?

Reply:
I don't think it's possible to define default values for properties in types. I think your syntax would something like: title can be any string or the string ''...

Reply:
IMHO we should leave it as required for know. We can be more precise later on. So you may just remove the empty string thing.

Reply:
Agree...